src/translations/en-US.json

```diff
@@ -856,5 +856,55 @@
         "value": "If the filter condition includes 0, blank values will be treated as 0 and included in calculations.",
         "comment": "Checkbox label tooltip. 'blank values' are data rows without any value (empty), which can be considered to be 0 during calculations. If the filter is set to '< 1' for example, it includes values of 0 (less than one) and therefore all the blank (empty) values are going to be considered to be 0 in calculations.",
         "limit": 0
+    },
+    "measureNumberFormat.inherit.optionLabel": {
```

Comment:
I believe this is not used anywhere anymore (inherit preset is not hardcoded anymore).